isis/src/control/apps/jigsaw/main.cpp

```diff
@@ -323,81 +323,136 @@ BundleSettingsQsp bundleSettings(UserInterface &ui) {
+                                                            positionVelocityAprioriSigma,
+                                                            positionAccelerationAprioriSigma);
+
+    // add all image observation numbers to this BOSS.
+    for (int sn = 0; sn < cubeSNs.size(); sn++) {
+      observationSolveSettings.addObservationNumber(cubeSNs.observationNumber(sn));
+    }
+
+    // append the GUI acquired solve parameters to BOSS list.
+    observationSolveSettingsList.append(observationSolveSettings);
   }
 
-  observationSolveSettings.setInstrumentPositionSettings(positionSolveOption,
-                                                          ui.GetInteger("SPKDEGREE"),
-                                                          ui.GetInteger("SPKSOLVEDEGREE"),
-                                                          ui.GetBoolean("OVERHERMITE"),
-                                                          positionAprioriSigma,
-                                                          positionVelocityAprioriSigma,
-                                                          positionAccelerationAprioriSigma);
 
   // If we are holding any images, then we need a BundleObservationSolveSettings for the held
   // images, and another one for the non-held images
   if (ui.WasEntered("HELDLIST")) {
     // Check that the held images are present in the input image list
     QString heldList = ui.GetFileName("HELDLIST");
-    QString fromList = ui.GetFileName("FROMLIST");
     SerialNumberList heldSNs(heldList);
-    SerialNumberList cubeSNs(fromList);
     checkImageList(heldSNs, cubeSNs);
 
+    double anglesAprioriSigma, angularVelocityAprioriSigma, angularAccelerationAprioriSigma;
+    anglesAprioriSigma = angularVelocityAprioriSigma = angularAccelerationAprioriSigma = Isis::Null;
+
+    double positionAprioriSigma, positionVelocityAprioriSigma, positionAccelerationAprioriSigma;
+    positionAprioriSigma = positionVelocityAprioriSigma = positionAccelerationAprioriSigma
+                          = Isis::Null;
+
     // The settings for the held images will have no pointing or position factors considered
     BundleObservationSolveSettings heldSettings;
-    BundleObservationSolveSettings::InstrumentPointingSolveOption noPointing = 
+    BundleObservationSolveSettings::InstrumentPointingSolveOption noPointing =
         BundleObservationSolveSettings::stringToInstrumentPointingSolveOption(
             "NoPointingFactors");
     heldSettings.setInstrumentPointingSettings(noPointing,
                                                 ui.GetBoolean("TWIST"),
                                                 ui.GetInteger("CKDEGREE"),
                                                 ui.GetInteger("CKSOLVEDEGREE"),
                                                 ui.GetBoolean("OVEREXISTING"),
-                                                positionAprioriSigma,
+                                                anglesAprioriSigma,
```

Comment:
Any particular reason this was changed? Just for consistency?

Reply:
This is a bug from before. The position sigmas should be passed into the position call and the angle sigmas should be passed into the angles call. Functionally this doesn't change anything other than output files because the image is held fixed so the sigmas are not used.